src/handbook/development/project-management.md

```diff
@@ -6,6 +6,12 @@ navTitle: Project Management
 
 This page details provides a comprehensive overview of the project management processes and procedures that guide product development at FlowFuse. 
 
+## Responsibilities
+
+The three roles primarily involved in product planning and development at FlowFuse are the CTO, Engineering Manager, and Product Lead. The Product Lead advances strategic priorities and develops the product roadmap to achieve those priorities, the CTO determines technical feasibility and identifies the scope and dependencies of roadmap items, and the Engineering Manager assigns tasks to the engineering team and is responsible for delivering work on schedule.
+
+The three meet weekly or bi-weekly to identify, define, discuss, and plan work for each release.
```

Comment:
Which is it, weekly or bi-weekly? The handbook should provide answers, not ask questions to the reader :)

Also, let's review if what's promised is being delivered on. Lastly - RACI wise, let's close the loop and ensure async the information is pushed back to other stakeholders.

```suggestion
The three meet bi-weekly to review progress of the current release and identify, define, discuss, and plan work for upcoming releases.
```